students/KellyM/Lesson3/list_lab.py

```diff
@@ -0,0 +1,113 @@
+"""Series 1
+    - Create a list that contains �Apples�, �Pears�, �Oranges� and �Peaches�.
+    - Display the list (plain old print() is fine�).
+    - Ask the user for another fruit and add it to the end of the list.
+    - Display the list.
+    - Ask the user for a number and display the number back to the user and the fruit corresponding to that number (on a 1-is-first basis). Remember that Python uses zero-based indexing, so you will need to correct.
+    - Add another fruit to the beginning of the list using �+� and display the list.
+    - Add another fruit to the beginning of the list using insert() and display the list.
+    Display all the fruits that begin with �P�, using a for loop.
+"""
+print("Series 1...")
+fruits = ['Apples','Pears','Oranges','Peaches']
+
+for item in fruits:
+    print(item)
+    
+question = input("Please enter the name of a fruit: ")
+fruits.append(question)
+print(fruits)
+
+number = int(input("Please enter a number: "))
+print('')
+print(fruits[number])
+print('')
```

Comment:
as mentioned in instructions, you would expect user to enter a number starting at 1, but indexes start at 0